EvolvSwiftSDK/EvolvSwiftSDK/Sources/Utils/OrderedDictionary.swift

```diff
@@ -0,0 +1,129 @@
+//
```

Comment:
Can you add tests for this file?

Reply:
yes. I will add them before our next meeting